cfg/2.0.0/definitions.yaml

```diff
@@ -7577,13 +7698,16 @@ groups:
       audit: "sshd -T | grep clientaliveinterval"   
       tests:
         test_items:
-        - flag: "ClientAliveInterval 300"
+        - flag: "clientaliveinterval"
```

Comment:
From the benchmark I think this needs to also test that `clientaliveinterval >= 1` (so I suppose that would be a bin_op test that tests both conditions are true with an `and`)

Reply:
seems right, I will fix it